### DISCUSSION TITLE
Automations: Trigger on group offline

### DISCUSSION THREAD

**Discussion by J-esper** [gameserverapp/Platform#999] [Suggestions]
Would love to be able to trigger automation or have a condition to check for group status = offline on the specific server. 
Players can be online on a different server, which is also a different group and that makes both groups show online. 

**For example in the current situation:**

> Group _G1_ exists on Server _S1_ with players _P1_ and _P2_.
> Group _G2_ exists on Server _S2_ with players _P1_ and _P2_.
> _P1_ is offline.
> _P2_ is online on _S2_.
> 
> Dashboard shows:
> _G1_ = Online
> _G2_ = Online.

I would want the result for the condition/trigger in the above case to be G1 = Offline, G2 = Online, maybe as well reflected on the dashboard. So I can run a command on {group.id} of G1

**Comment by mvaessen** [comment]
hi, thanks for the suggestion.

In the example you provided, the right data would be:
> Dashboard shows:
> G1 = Offline
> G2 = Online.

As your example, Player 2 is online on server 2. So G1 is not online.

**Reply by J-esper** [reply]
Random example I pulled, in this case both groups have the same name, but it's also true when they have different names.
![image](https://github.com/user-attachments/assets/2adb2c90-2416-4581-ae67-fbd173661729)

**Reply by mvaessen** [reply]
im only seeing a screenshot of the members for 1 group. Did you check the members for the other group?

**Reply by J-esper** [reply]
![image](https://github.com/user-attachments/assets/9fb00fe2-3f89-4381-8b52-a62b35698435)

**Reply by mvaessen** [reply]
so both groups have online members, right?

in that case both groups will show up as being online, as both have online members.

**Reply by J-esper** [reply]
But one group is for TheCenter server, the other TheIsland server. It's the same players, but their character only exists on TheIsland currently, so I'd expect the group for TheCenter to be offline and TheIsland online.

**Reply by mvaessen** [reply]
ah, thanks for the extra info.

That sounds more like a bug, so its best to report that in the issue section. You can leave the actual suggestion on here
